tests/std/tests/GH_000685_condition_variable_any/test.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// Test GH#685 "wait_for in condition_variable_any should unlock and lock"

```

Comment:
We conventionally use dashes in product/test code comments when referring to GitHub issues and other bugs, for consistency with GitHub autolinking (although it isn't active in the code itself).

```suggestion
// Test GH-685 "wait_for in condition_variable_any should unlock and lock"
```